### DISCUSSION TITLE
[Feature Request] Webhooks

### DISCUSSION THREAD

**Discussion by ados8** [donetick/donetick#99] [Ideas]
With the self hosting option will you be supporting webhooks?
This will allow users running their own messaging services to use Webhooks for notifications instead of Telegram or SMTP.

**Comment by meauxt** [comment]
Thanks to @dkhalife for taking a look at this! We'll evaluate if this is the right way to move forward.

**Reply by ados8** [reply]
If Webhooks are used not only is it one of the most widely supported messaging protocols but users can then combine with n8n to have effectively any receiving messaging endpoint they want without the need to develop for 20+ platforms.
